### PR TITLE
updated to pcl 1.9.1

### PR DESCRIPTION
**Pull request by ffurrer**

### CODE REVIEW

File: pcl_catkin/CMakeLists.txt [hunk 1]
```diff
@@ -27,19 +27,24 @@ SET(PCL_CXX_FLAGS "-Wall -Wextra -Wno-unknown-pragmas -fno-strict-aliasing -Wno-
...
 
 cs_add_library(${PROJECT_NAME} src/dependency_tracker.cc)
 add_dependencies(${PROJECT_NAME} pcl_src)
 
-cs_install()
+install(DIRECTORY ${CATKIN_DEVEL_PREFIX}/lib/
+        DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+      FILES_MATCHING PATTERN "libpcl*")
+
+# cs_install()
```
Comment (ffurrer) [inline pcl_catkin/CMakeLists.txt]: not sure anymore why I had to do this :-S

File: pcl_catkin/CMakeLists.txt [hunk 2]
```diff
@@ -27,19 +27,24 @@ SET(PCL_CXX_FLAGS "-Wall -Wextra -Wno-unknown-pragmas -fno-strict-aliasing -Wno-
 SET(PCL_CXX_FLAGS "${PCL_CXX_FLAGS} -std=c++11")
 
 ExternalProject_Add(pcl_src
-  GIT_REPOSITORY  https://github.com/ethz-asl/pcl
-  GIT_TAG pcl_1.8.0_devel
+  GIT_REPOSITORY  https://github.com/PointCloudLibrary/pcl
+  GIT_TAG pcl-1.9.1
```
[the remark this answers is not in the document]
Comment (ffurrer) [reply pcl_catkin/CMakeLists.txt]: looks like

File: pcl_catkin/CMakeLists.txt (hunk 1 above)
Comment (ffurrer) [reply pcl_catkin/CMakeLists.txt]: I can try with the default cs_install again, I was fiddling around with this for it to build and then was to lazy to retry with this once it build ;). Will try this weekend though.

Comment (ffurrer) [reply pcl_catkin/CMakeLists.txt]: yup, seems to work now, the issues were due to the submodules not depending on pcl_catkin but system pcl.

**Comment by mfehr** [comment]
test this please (created new jenkins job for 14.04, 16.04, 18.04, osx)

**Comment by ffurrer** [comment]
test this please

**Comment by mfehr** [comment]
test this please (changed jenkins config)

**Comment by mfehr** [comment]
test this please (switched off broken hyrax-xenial jenkins node)

**Comment by mfehr** [comment]
@margaritaG  @ffurrer Found the gtest issue. Now there are only these hztests that are failing and I found the reason for it as well, our [jenkins configuration always runs unit tests with -j1](https://github.com/ethz-asl/continuous_integration/blob/master/catkin_make_repo#L64), which makes the hztests extremely flaky. This can be reproduced on the local machine if you use the same command. I think we should modify the continuous_integration scripts to have a flag that can disable this. I might open a PR next week for this, or not, depending on my xmas spirit :)

**Comment by mfehr** [comment]
As discussed with @ffurrer we will merge this now and adapt jenkins later. The only remaining issue is the flaky hz tests. Here it goes.

**Pull request merged**